op-batcher/metrics/metrics.go

```diff
@@ -326,8 +341,14 @@ func estimateBatchSize(block *types.Block) uint64 {
 		if tx.IsDepositTx() {
 			continue
 		}
-		// Add 2 for the overhead of encoding the tx bytes in a RLP list
-		size += tx.Size() + 2
+		bigSize := types.EstimatedL1SizeScaled(tx.RollupCostData())
+		bigSize = bigSize.Div(bigSize, big.NewInt(1e6)) // unscale it
+		if bigSize.IsUint64() {
+			size += bigSize.Uint64()
+		} else {
+			// this shouldn't ever happen
+			size += tx.Size() + 2
+		}
```

Comment:
this change will change the metrics pretty significantly; should we track the compressed size separately?

also should we pass the entire block to the compression estimation?

Reply:
Yeah it's going to be like 2/3rds lower I guess. I couldn't really think of a great reason we'd want to track uncompressed bytes pending though. Interesting idea to pass the entire block. Unfortunately the function was tuned per-tx, so it might make things less accurate?

Reply:
I think we should avoid changing the scale of the existing metric, and instead add a new one that is a compressed-bytes-estimate metric

Reply:
how about changing the name and removing the old one?  I guess I'm assuming nobody is really using it, but I could be wrong.

Reply:
changed to just be a new parameter leaving old ones intact.